src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -442,20 +445,23 @@ FixedwingPositionControl::get_auto_airspeed_setpoint(const hrt_abstime &now, con
 
 	float airspeed_min_adjusted = _param_fw_airspd_min.get();
 
-	/*
-	 * Calculate accelerated stall airspeed factor from commanded bank angle and use it to increase minimum airspeed.
-	 *
-	 * Increase lift vector to balance additional weight in bank
-	 * cos(bank angle) = W/L = 1/n, n is the load factor
-	 *
-	 * lift is proportional to airspeed^2 so the increase in stall speed is Vsacc = Vs * sqrt(n)
-	 */
+	float load_factor = 1.0f;
 
-	if (_airspeed_valid && PX4_ISFINITE(_att_sp.roll_body)) {
-		airspeed_min_adjusted = constrain(_param_fw_airspd_stall.get() / sqrtf(cosf(_att_sp.roll_body)),
-						  airspeed_min_adjusted, _param_fw_airspd_max.get());
+	if (PX4_ISFINITE(_att_sp.roll_body)) {
+		load_factor = 1.0f / sqrtf(cosf(_att_sp.roll_body));
 	}
 
+	float weight_ratio = 1.0f;
+
+	if (_param_weight_base.get() > FLT_EPSILON && _param_weight_gross.get() > FLT_EPSILON) {
+		weight_ratio = math::constrain(_param_weight_gross.get() / _param_weight_base.get(), 0.5f, 2.0f);
```

Comment:
can we make `0.5` and `2.0` constants? e.g. `MIN_WEIGHT_RATIO`, `MAX_WEIGHT_RATIO` with description of what they are bounding